SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSchemaGenerator.h

```diff
@@ -12,27 +12,27 @@ namespace SpatialGDKEditor
 	{
 		SPATIALGDKEDITOR_API bool IsSupportedClass(const UClass* SupportedClass);
 
-		SPATIALGDKEDITOR_API TSet<UClass*> GetAllSupportedClasses();
+		SPATIALGDKEDITOR_API TSet<UClass*> GetAllSupportedClasses(const TArray<UObject*>& AllClasses);
 		
 		SPATIALGDKEDITOR_API bool SpatialGDKGenerateSchema();
```

Comment:
I'm considering creating a follow up task for the functions here that aren't yet tested. I think it's a good opportunity for someone to pair with you, and learn how these tests work.

Do you think that would be worthwhile?

Reply:
Yes, that could be worthwhile.

Reply:
UNR-2298